README.rst

```diff
@@ -1,2 +1,54 @@
 This package provides an ExplainCollection class
-that allows common MongoDB commands to be explained
\ No newline at end of file
+that allows that allows PyMongo's Collection methods to be explained.
+
+
+Tutorial
+########
+The intended use case for this package is to debug ``pymongo`` commands.
+This can be done by simply swapping out ``Collection`` for ``ExplainCollection``,
+which has the same methods but will run explain on them *instead* of executing them.
+The first step is to create a ``MongoClient`` instance::
+
+    client = MongoClient('mongodb://localhost:27017/')
+
+Once this is done, you can simply get a collection instance, and then wrap it in the ``ExplainCollection`` class.::
+
+    collection = client.db.products
+    explain = ExplainCollection(collection)
+
+Now you are ready to run some commands. Remember that these commands will not be executed, they will simply have explain
```

Comment:
*now you are ready to **explain** some commands

Reply:
done.